### PR TITLE
Add the Jinja template for Amazon ECS

### PR DESCRIPTION
**Pull request by tmorrisonsfx**
Jira issue: https://signalfuse.atlassian.net/browse/DOCS-1388

Docs URL: [Amazon EC2 Container Service (ECS)](https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.amazon.ec2.container.service.(ecs).html)

What changed?
- Added the README.md.jinja and update the meta.yaml
- Added the content from the ECS Deployment README to the tile
- Added the content from the ECS Deployment Guide and Fargate Deployment Guide directly to the tile
- Deleted the README.md file

### CODE REVIEW

**Comment by tmorrisonsfx** [comment]
@pohannigansfx @keitwb This is my first test of the new template system. PTAL.

File: aws-ecs/README.md.jinja [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+1. Select **Task Definitions**.
+2. Click **Create new Task Definition**.
+3. Select **EC2** and click **Next step**. The agent only supports EC2 mode and not Fargate at this time.
+4. Scroll to the bottom of the page and click **Configure via JSON**.
+5. Paste in the contents of the signalfx-agent-task.json.
+6. Click **Save.**
+7. Click the **signalfx-agent** container definition under **Container Definitions** and find the section on environment variables.
+8. Change the value of the `ACCESS_TOKEN`, `INGEST_URL`, `API_URL`, and `TRACE_ENDPOINT_URL` environment variables to the access token for your SignalFx organization and endpoints you want to send metrics to.
+9. Click **Update** and then click **Create** at the bottom of the task definition input form.
```
Comment (pohannigansfx) [inline aws-ecs/README.md.jinja:50]: two actions in the same step rather than in two separate steps?

File: aws-ecs/README.md.jinja [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+6. Click **Next step**.
+7. Choose the default setting for the remaining options.
+8. Click **Next step**.
+9. Choose the default setting for the remaining options.
+10. Click **Next step**.
+11. Click **Create Service**. The agent is deployed onto each node in the Amazon ECS cluster, and the infrastructure and docker metrics start flowing soon.
+
+##### Step 3: Configuration
+
+The main technique for configuring the agent is to have a configuration file downloaded from the network using curl in the agent container's initialization script. By default, the agent pulls from the <a href="https://github.com/signalfx/signalfx-agent/blob/master/deployments/ecs/agent.yaml" target="_blank">configuration file</a> in the GitHub repository that provides a basic configuration that might suffice for basic monitoring cases. If you wish to provide a more complex configuration file, you can set the `CONFIG_URL` environment variables in the agent task definition to the URL of the configuration file. This location must be accessible from the Amazon ECS cluster.
```
Comment (pohannigansfx) [inline aws-ecs/README.md.jinja:82]: This para includes "if you wish..." but Splunk usage dictionary says to scrub "wish" and use "want" instead.

**Comment by tmorrisonsfx** [comment]
Closing in favor of https://github.com/signalfx/integrations/pull/870.

**Pull request closed without merging**